ffi/diplomat/c/examples/fixeddecimal_tiny/Makefile

```diff
@@ -27,7 +27,7 @@ icu_capi_staticlib_tiny/target/x86_64-unknown-linux-gnu/release-opt-size/libicu_
 	RUSTFLAGS="-Clinker-plugin-lto -Clinker=$(CLANG) -Ccodegen-units=1 -Clink-arg=-flto -Cpanic=abort" cargo +${ICU4X_NIGHTLY_TOOLCHAIN} panic-abort-build --target x86_64-unknown-linux-gnu --profile=release-opt-size
 
 decimal-bn-en.postcard:
-	cargo run -p icu_datagen -- --locales en bn --keys "decimal/symbols@1" --cldr-root ../../../../../provider/testdata/data/cldr/ --format blob --out decimal-bn-en.postcard
+	cargo run -p icu_datagen -- --locales en bn --keys "decimal/symbols@1" --format blob --out decimal-bn-en.postcard
```

Comment:
Nit (optional): I prefer to avoid using source data as much as possible; it was fine when it was pointing to the repo data.

Reply:
The FFI CI already uses live data in other places though.

Reply:
This makes a better example.